src/wing/CCPACSWingCell.cpp

```diff
@@ -755,30 +769,28 @@ TopoDS_Shape CCPACSWingCell::GetRibCutGeometry(std::pair<std::string, int> ribUi
     return cutGeometry.shape;
 }
 
-bool CCPACSWingCell::IsPartOfCell(TopoDS_Face f)
+template<class T>
+bool CCPACSWingCell::IsPartOfCellImpl(T t)
 {
     Update();
 
     Bnd_Box bBox1, bBox2;
     BRepBndLib::Add(m_geometryCache.value().cellSkinGeometry, bBox1);
-    TopoDS_Face face = TopoDS::Face(m_parent->GetParentElement()
+    TopoDS_Edge edge = TopoDS::Edge(m_parent->GetParentElement()
```

Comment:
i believe we should have TopoDS_Shape here, as the transformed shape "t" can be an edge or a face.

Reply:
Thanks! This was indeed a bug.